### PR TITLE
DCOS_OSS-3554 Correcting .tmpl file with incorrect code at transport_…

### PR DESCRIPTION
**Pull request by SarahStegall**
…encryption json file.

## Description
https://jira.mesosphere.com/browse/DCOS_OSS-3554

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

Edited 

services/include/security-configure-transport-encryption.tmpl

to add

"security": {
"transport_encryption": {
"enabled": <true|false default false>,

 

This will change all versions that call this template:

2.3.0-1.0.0

2.2.0-1.0.0

2.1.0-1.0.0

### CODE REVIEW

File: pages/services/include/security-configure-transport-encryption.tmpl [hunk 1]
```diff
@@ -26,7 +26,7 @@ Install the DC/OS {{ model.techName }} service including the following options i
         "service_account_secret": "<full path of service secret>",
         "security": {
             "transport_encryption": {
-                "enabled": true{{ model.security.plaintext }}
```
Comment (benclarkwood) [inline pages/services/include/security-configure-transport-encryption.tmpl]: Some services support plaintext (which is why this template was there). 

What's the issue that was occurring?

Comment (benclarkwood) [reply pages/services/include/security-configure-transport-encryption.tmpl]: Ideally we'd be able to just correct the template such that the formatting works out right.

@nickbp @elezar have either of you had better luck with special symbols in a template?

Comment (nickbp) [reply pages/services/include/security-configure-transport-encryption.tmpl]: Slathering it with triple brackets may help: `{{{foo}}}`

**Comment by SarahStegall** [comment]
This template does not need to be corrected. I am deleting the PR and will fix this issue another way. Closing without merge.

**Pull request closed without merging**